content/guides/trash/permanently-delete-a-file.md

```diff
@@ -0,0 +1,19 @@
+---
+rank: 2
+related_endpoints:
+ - delete_files_id
+related_guides: []
+required_guides: []
+related_resources:
+  - files
+---
+
+# Permanently delete a file
```

Comment:
Style note: I've been trying to keep all H1 headers in Title Case and without using `a` and `the`. So in this case: 

```
# Permanently Delete File
```

We can change this later if we want to.